packages/reference/src/components/LinkActions/helpers.ts

```diff
@@ -46,7 +46,7 @@ export async function selectMultipleEntities(props: {
   sdk: FieldExtensionSDK;
   entityType: EntityType;
   editorPermissions: EditorPermissions;
-}) {
+}): Promise<(Entry | Asset)[] | null> {
```

Comment:
Is it necessary to explicitly type here?

Reply:
Indeed no, I just removed it, I needed to add this explicit return because when I changed the shape of Entry by adding the `metadata.tags[]` property the type inferral was not working as expected, but I updated the Asset's interface after, and that fixed the type inferral issue that I had before 👍